src/pages/onboarding/onboarding.jsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+import "./onboarding.css";
+import { AiFillEdit } from "react-icons/ai";
+import { getGreet, getHour, getMinute } from "../../utils";
+import { getQuote, getWeather } from "../../services";
+import { Weather } from "../../components";
+
+export const OnBoarding = () => {
+  const [userName, setUserName] = useState("");
+  const [printUserName, setPrintUserName] = useState("");
```

Comment:
You can use one state for the same purpose

Reply:
one state is used for username and another state is used  when button is clicked